server/database/db_build.sql

```diff
@@ -0,0 +1,53 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users cascade;
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  user_name VARCHAR(100) NOT NULL,
+  guid UNIQUEIDENTIFIER PRIMARY KEY default NEWID() ,
+  phone VARCHAR(12),
+  birth_date DATE,
+  email VARCHAR(30),
+  university VARCHAR(30),
+  address VARCHAR(100),
+  role VARCHAR(30),
+  profession VARCHAR(30),
+  password VARCHAR(30),
```

Comment:
using fix min number of VARCHAR , using min 100 , to avoid any error 
address , profession  make 500
convert password to Text  ( password TEXT NOT NULL  ) , becomes the password must be hash